js/models/configModel.js

```diff
@@ -1,6 +1,7 @@
 import Adapt from 'core/js/adapt';
 import wait from 'core/js/wait';
 import LockingModel from 'core/js/models/lockingModel';
+import logging from '../logging';
```

Comment:
Minor: better to use the non-relative filepath?
```suggestion
import logging from 'core/js/logging';
```